compiler/controllers.py

```diff
@@ -81,6 +81,12 @@ def compile(request_data: MultiDict, token: str, session: Session,
 
     # We don't want to compile the same source package twice.
     force = request_data.get('force', False)
+
+    # Support label and link for PS/PDF Stamping
+    # Test
+    stamp_label = request_data.get('stamp_label', None) # or is '' better?
```

Comment:
I think using ``None`` is clearer; see suggestions on ``compiler.py``.

Reply:
Great!